### PR TITLE
influxdb 2.0.8

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Comment by cho-m** [comment]
It's using a GNU command `sha256sum`, so may need to use `coreutils` to build.
```
100 69.0M  100 69.0M    0     0  9344k      0  0:00:07  0:00:07 --:--:-- 12.5M
./scripts/fetch-ui-assets.sh: line 30: sha256sum: command not found
Checksums did not match for downloaded UI assets!
```

File: Formula/influxdb.rb [hunk 1]
```diff
@@ -52,10 +51,10 @@ def install
     ENV.prepend_path "PATH", buildpath/"bootstrap"
 
     # Extract pre-build UI resources to the location expected by go-bindata.
-    resource("ui-assets").stage(buildpath/"ui/build")
+    resource("ui-assets").stage(buildpath/"static/data/build")
 
     # Embed UI files into the Go source code.
-    system "make", "generate"
+    system "make", "generate-web-assets"
```
Comment (cho-m) [inline Formula/influxdb.rb:57]: May need to check if this works correctly. Even creating an empty directory `./static/data/build` will successfully build.

`make generate` still exists, but also runs `make generate-sources`, which requires some extra Go tools like github.com/benbjohnson/tmpl, golang.org/x/tools/cmd/stringer, github.com/gogo/protobuf/protoc-gen-gogo, ...

`make generate-web-assets` is now used by v2.0.8 goreleaser https://github.com/influxdata/influxdb/blob/v2.0.8/.goreleaser.yml#L57 (`make generate` was used  in v2.0.7)

**Comment by carlocab** [comment]
CC @danxmoran, in case you have time to take a look

File: Formula/influxdb.rb (hunk 1 above)
Comment (danxmoran) [reply Formula/influxdb.rb:57]: AFAICS this works. To test I checked out this PR, reinstalled the formula from source, ran the server, and opened the login page. The UI shows version 2.0.8 and front-end commit `cb0a87e`, which is what I'd expect for the referenced UI version.

(There seems to be an unrelated problem with browser-side caching of the version info, but I think that's unrelated to Homebrew).

**Comment by danxmoran** [comment]
LGTM! Thanks @chenrui333

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**